src/TodoList.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemSecondaryAction from "@material-ui/core/ListItemSecondaryAction";
+import ListItemText from "@material-ui/core/ListItemText";
+import Checkbox from "@material-ui/core/Checkbox";
+import IconButton from "@material-ui/core/IconButton";
+import DeleteIcon from "@material-ui/icons/Delete";
+
+const TodoList = ({ todos, deleteTodo }) => (
+  <List>
+    {todos.map((todo, index) => (
+      <ListItem key={index.toString()} dense button>
+        <Checkbox tabIndex={-1} disableRipple />
```

Comment:
When the checkbox is selected, the text should also be striked through, right?

Reply:
yes